src/lib.rs

```diff
@@ -11,7 +11,7 @@ mod raw_data;
 pub use error::TdmsError;
 pub use file::TdmsFile;
 pub use file::TdmsFileWriter;
-pub use io::data_types::TdmsStorageType;
+pub use io::data_types::{TdmsStorageType, DataType};
```

Comment:
@JamesMc86 I made `DataType` public so that I could use it in my high-level tests. Not sure if this is how you want to expose supported datatypes in the API.